docs/sources/mimir/configure/configure-object-storage-backend.md

```diff
@@ -145,6 +161,46 @@ ruler_storage:
     container_name: mimir-ruler
 ```
 
+#### Azure Workload Identity
+
+Here is an example config for using Azure Workload Identity.
+
+```yaml
+---
+common:
+  storage:
+    backend: azure
+    azure:
+      account_name: mimirprod
+      endpoint_suffix: "blob.core.windows.net"
+
+blocks_storage:
+  azure:
+    container_name: mimir-blocks
+
+alertmanager_storage:
+  azure:
+    container_name: mimir-alertmanager
+
+ruler_storage:
+  azure:
+    container_name: mimir-ruler
+
+serviceAccount:
+  create: true
+  name: mimir-storage
+  annotation:
+    "azure.workload.identity/use: "true"
+  labels:
+    "azure.workload.identity/use: "true"
+
+global:
+  podlabels:
+    "azure.workload.identity/use: "true"
+```
+
+_note: federated token is not supported with Mimir unlike with Tempo_
```

Comment:
```suggestion
{{< admonition type="note" >}}
Unlike with Tempo, federated tokens are not supported with Mimir.
{{< /admonition >}}
```

Reply:
I think we should use our standard format for admonitions per https://grafana.com/docs/writers-toolkit/write/shortcodes/#admonition